src/app/components/Drbb/DrbbResult.jsx

```diff
@@ -19,10 +19,10 @@ const DrbbResult = (props) => {
   if (!work || !work.uuid || !work.title) return null;
 
   const {
-    agents,
     title,
     editions,
   } = work;
+  const authors = work.authors ? work.authors : work.agents.filter(agent => agent.roles.includes('author'));
```

Comment:
Are we sure there cannot be both `authors` and `agents`?

Reply:
Good question. The `authors` prop is for API v4. In v3, authors were given in an `agents` property (with a `role` of `"author"`). This supports both. I added an inline comment to document this.